src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Standards/WhenHandlingGetStandardVersionsByOrganisationIdAndStandardReferenceRequest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models;
+using SFA.DAS.AssessorService.Api.Types.Models.AO;
+using SFA.DAS.AssessorService.Application.Handlers.ao;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.UnitTests.Handlers.Standards
```

Comment:
Just need to change the unit test class and file name to match new GetAppliedStandardVersionsForEPAO

Reply:
Fixed